Formula/inspircd.rb

```diff
@@ -18,13 +18,22 @@ class Inspircd < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "argon2"
+  depends_on "gnutls"
+  depends_on "libpq"
+  depends_on "mysql-client"
+
+  uses_from_macos "openldap"
 
   skip_clean "data"
   skip_clean "logs"
 
   def install
-    system "./configure", "--enable-extras=ldap"
-    system "./configure", "--prefix=#{prefix}"
+    system "./configure", "--enable-extras",
+                          "argon2 ldap mysql pgsql regex_posix regex_stdlib ssl_gnutls sslrehashsignal"
+    system "./configure", "--disable-auto-extras",
+                          "--distribution-label", "homebrew-#{revision}",
```

Comment:
`revision` here currently returns `0`. Is that your intention with this?

Reply:
Yes. It's for uniquely identifying the revision of the package with which the software was built.